Userland/Libraries/LibWeb/HTML/Canvas/CanvasTransform.h

```diff
@@ -66,6 +76,25 @@ class CanvasTransform {
         transform(a, b, c, d, e, f);
     }
 
+    // https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-settransform-matrix
+    WebIDL::ExceptionOr<void> set_transform(Geometry::DOMMatrix2DInit& init)
+    {
+        // 1. Let matrix be the result of creating a DOMMatrix from the 2D dictionary transform.
+        auto& realm = static_cast<IncludingClass&>(*this).realm();
+        auto maybe_matrix = Geometry::DOMMatrix::create_from_dom_matrix_2d_init(realm, init);
+        if (maybe_matrix.is_exception())
+            return maybe_matrix.exception();
+        auto matrix = maybe_matrix.release_value();
```

Comment:
can't this be TRY()'d ?

Reply:
Oh really nice! I didn't know that you could use TRY with ExceptionOr. Now that I know this I have added a fix for an older commit of my to also use TRY in DOMMatrix and DOMPointReadOnly.